symfony/README.md

```diff
@@ -0,0 +1,35 @@
+# About Algolia's Symfony bundle
+
+Algolia Symfony bundle was created by, and is maintained by [Algolia](https://github.com/algolia).
+
+## Get started locally
+> requires :
+* [Composer](https://getcomposer.org/)
+* [PHP 7.1.3+](https://www.php.net/releases/index.php)
+
+```bash
+# Install from Algolia's Playground from Github
+git clone https://github.com/algolia/api-clients-playground
+
+# Move to Laravel-scout-extended folder.
+cd api-clients-playground/symfony
+
+# Install dependencies
+composer install
+
+# Add your credential in .env
+ALGOLIA_APP_ID='YOUR APPLICATION ID'
+ALGOLIA_SECRET='YOUR ADMIN API KEY'
+
+# Your database doesn't exists, so you just need to create one and load data then send it to Algolia
+composer update
```

Comment:
You shouldn't do things by magic. The `composer update` is for updating dependencies and not for `Your database doesn't exists, so you just need to create one and load data then send it to Algolia`.

You should ask for people to do those commands.
```
php bin/console make:migration
php bin/console doctrine:migrations:migrate
php bin/console doctrine:fixtures:load
php bin/console search:import
```

and remove those comands from composer.json.